src/test/java/mops/termine2/services/GruppeServiceTest.java

```diff
@@ -1,181 +1,181 @@
-		
-		List<Gruppe> expected = new ArrayList<Gruppe>(
-			Arrays.asList(
-				g1, g2
-			));
-		
-		List<Gruppe> result = gruppeService.sortGroupsByName(input);
-		
-		assertThat(result).isEqualTo(expected);		
-	}
-	
-	@Test
-	public void testAccessDeniedNull() {
-		Account account = new Account("studentin", "abc@def.de", null, null);
-		boolean result = gruppeService.checkGroupAccessDenied(account, null);
-		assertThat(result).isEqualTo(false);
-	}
-	
-	@Test
-	public void testAccessDeniedMinus1() {
-		Account account = new Account("studentin", "abc@def.de", null, null);
-		boolean result = gruppeService.checkGroupAccessDenied(account, -1L);
-		assertThat(result).isEqualTo(false);
-	}
-	
-	@Test
-	public void testAccessDenied() {
-		Account account = new Account("studentin", "abc@def.de", null, null);		
-		when(benutzerGruppeRepository.findByBenutzerAndGruppeId("studentin", 1L))
-			.thenReturn(null);
-		
-		boolean result = gruppeService.checkGroupAccessDenied(account, 1L);
-		
-		assertThat(result).isEqualTo(true);
-	}
-	
-	@Test
-	public void testAccessNotDenied() {
-		Account account = new Account("studentin", "abc@def.de", null, null);
-		BenutzerGruppeDB user = new BenutzerGruppeDB();
-		user.setBenutzer("studentin");
-		user.setGruppe("Test");
-		user.setGruppeId(1L);
-		user.setId(1L);
-		when(benutzerGruppeRepository.findByBenutzerAndGruppeId("studentin", 1L))
-			.thenReturn(user);
-		
-		boolean result = gruppeService.checkGroupAccessDenied(account, 1L);
-		
-		assertThat(result).isEqualTo(false);
-	}
-	
-}
+package mops.termine2.services;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import mops.termine2.authentication.Account;
+import mops.termine2.database.BenutzerGruppeRepository;
+import mops.termine2.database.entities.BenutzerGruppeDB;
+import mops.termine2.models.Gruppe;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+public class GruppeServiceTest {
+	
+	private transient GruppeService gruppeService;
+	
+	private transient BenutzerGruppeRepository benutzerGruppeRepository;
+	
+	@BeforeEach
+	public void setUp() {
+		benutzerGruppeRepository = mock(BenutzerGruppeRepository.class);
+		gruppeService = new GruppeService(benutzerGruppeRepository);
+	}
+	
+	@Test
+	public void testLoadByBenutzer() {
+		Account account = new Account("studentin", "abc@def.de", null, null);
+		List<String> groupNames = new ArrayList<>(Arrays.asList("Best group eva", "FIXME", "Last One :("));
+		
+		List<BenutzerGruppeDB> gruppenDB = new ArrayList<>();
+		List<Gruppe> gruppen = new ArrayList<>();
+		for (String name : groupNames) {
+			BenutzerGruppeDB gruppeDB = new BenutzerGruppeDB();
+			gruppeDB.setGruppe(name);
+			gruppenDB.add(gruppeDB);
+			
+			Gruppe gruppe = new Gruppe();
+			gruppe.setName(name);
+			gruppen.add(gruppe);
+		}

```

Comment:
In diesem Test werden die IDs nicht gesetzt, somit wird am Ende des Tests null mit null verglichen. Allerdings sollten hier tatsächlich richtige IDs gesetzt werden, damit die Logik korrekt getestet wird.
Die Zeilen hast du zwar nicht verändert, aber sie sind relevant zu deinem Pull Request.